### PR TITLE
[sp] add graphs to sidekick view

### PR DESCRIPTION
**Pull request by shrey-mage**
# Summary
- add "Data completion", "Number of unique values", "Distribution of values", and "Correlations" charts to sidekick view
- populate graph data from backend
- update dark theme chart colors
- update various charts to use colors based on theme context

### (Styling is a WIP and will be updated soon)

# Tests
image
image

cc: @johnson-mage @tommydangerous @dy46

### CODE REVIEW

File: mage_ai/frontend/components/Sidekick/index.tsx [hunk 1]
```diff
@@ -146,9 +148,11 @@ function Sidekick({
         </Spacing>
       }
       {activeView === ViewKeyEnum.GRAPHS &&
-        <Text>
-          Graphs
-        </Text>
+        <BlockOverview
```
Comment (johnson-mage) [inline mage_ai/frontend/components/Sidekick/index.tsx:151]: `BlockCharts` or `BlockVisualizations` seems like a more descriptive component name.

**Comment by johnson-mage** [comment]
@shrey-mage I know you mentioned styling is WIP, so im assuming you're going update the border color to not be white (I used `#1D1D20` in the reports table). And the black font is hard to see on the correlations matrix so a muted color may work better.

**Review by johnson-mage — APPROVED** [review]
approved after addressing comments

File: mage_ai/frontend/components/Sidekick/index.tsx (hunk 1 above)
Comment (shrey-mage) [reply mage_ai/frontend/components/Sidekick/index.tsx:151]: you're right, the idea was to match the `Overview` component but that's better clarity-wise

**Comment by shrey-mage** [comment]
> @shrey-mage I know you mentioned styling is WIP, so im assuming you're going update the border color to not be white (I used `#1D1D20` in the reports table). And the black font is hard to see on the correlations matrix so a muted color may work better.

I switched to using `RowDataTable`, should be more in line with the reports tab's styling now:

image

**Pull request merged**